ext/lvgl/lvgl.lb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Raphael Lehmann
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":lvgl"
+    module.description = """
+# LVGL graphics library
+
+LVGL is a free and open-source graphics library providing everything you need
+to create embedded GUI with easy-to-use graphical elements, beautiful visual effects
+and low memory footprint.
+
+- https://lvgl.io/
+- https://github.com/lvgl/lvgl
+"""
```

Comment:
Ah, we should mention the `lv_conf_local.h` mechanism, I'll add a fixup commit in a second.

Reply:
Hm, there's a file called `lv_conf_internal.h` that contains a *lot* of defaults.

We should also implement `LV_TICK_CUSTOM_SYS_TIME_EXPR` to use `modm::Clock::now` to remove the need for a manual call to `lv_tick_inc()`.

Reply:
Ok, I've implemented and tested `LV_TICK_CUSTOM_SYS_TIME_EXPR`.

Reply:
The only other thing I found interesting in terms of a modm port was `LV_ATTRIBUTE_FAST_MEM` which we could set to `__attribute__((sections(".fastcode")))`. However, 15-20kB of RAM seems like a lot and would probably cause issues if set by default.

```
/* Prefix performance critical functions to place them into a faster memory (e.g RAM)
 * Uses 15-20 kB extra memory */
#define  LV_ATTRIBUTE_FAST_MEM
```